forem/forem
# Organisation members page to show all users

## What type of PR is this? (check all applicable)


## Description

This PR introduces new `/members` page on organisation page which shows all users of that org.

There are two major changes:
1. Create new `/members` page on org page
2. Update org page sidebar to limit the users to `50` and show `See All Members` button at bottom.


## Related Tickets & Documents



- Closes #19511

## QA Instructions, Screenshots, Recordings

Designs - [Figma link](https://www.figma.com/file/9UfSjNYJs745w3Q8JCUqsL/Small-Wins---%E2%80%A8Core-Experience-%E2%9A%A1%EF%B8%8F?type=design&node-id=2080-9066&t=2mctDCv5ltqWSjth-4)

**How to test?**
1. First make a change in `app/controllers/stories_controller.rb`, change `user_limit` from 50 to 2 or 3 so that you can easily test it.
2. Now visit any organisation page which has more than the limit and notice `See All Members` button, click on it to visit `/members` page.

**Output**

| Light  | Dark |
| ------------- | ------------- |
| <img width="303" alt="Screenshot 2023-06-16 at 3 05 25 PM" src="https://github.com/forem/forem/assets/9396084/b2bc9eef-2c31-47c9-afe4-c362c0b5b2f7"> | <img width="311" alt="Screenshot 2023-06-16 at 3 05 10 PM" src="https://github.com/forem/forem/assets/9396084/2d43883f-c5d4-4493-bc80-d03e546b914b"> |
| <img width="1512" alt="Screenshot 2023-06-16 at 3 05 38 PM" src="https://github.com/forem/forem/assets/9396084/dbde0c2f-ae75-4cf6-850f-a5850cf7b358"> | <img width="1512" alt="Screenshot 2023-06-16 at 3 05 01 PM" src="https://github.com/forem/forem/assets/9396084/62ed0452-263d-439d-8062-e73f4dbd2a9d"> |

<img width="877" alt="Screenshot 2023-06-23 at 1 17 35 PM" src="https://github.com/forem/forem/assets/9396084/a3ed9c36-1f6d-44c4-ab74-3e31addd0644">
<img width="804" alt="Screenshot 2023-06-23 at 1 17 26 PM" src="https://github.com/forem/forem/assets/9396084/a1d8b1d0-7f7a-4ed3-a308-0bf84083845e">
<img width="791" alt="Screenshot 2023-06-23 at 1 17 19 PM" src="https://github.com/forem/forem/assets/9396084/031abde5-13f6-4574-91f3-23a8d4370b47">
<img width="449" alt="Screenshot 2023-06-23 at 1 17 11 PM" src="https://github.com/forem/forem/assets/9396084/727aec98-4773-4ebf-8dff-de434d97f56c"> <img width="410" alt="Screenshot 2023-06-23 at 1 17 03 PM" src="https://github.com/forem/forem/assets/9396084/651bca2c-66e9-4062-8fb6-6c818ce18cef"> <img width="348" alt="Screenshot 2023-06-23 at 1 16 55 PM" src="https://github.com/forem/forem/assets/9396084/3a1b783d-9e26-4960-ab13-c14f3134f6c0">


### UI accessibility checklist
_If your PR includes UI changes, please utilize this checklist:_

<img width="1387" alt="Screenshot 2023-06-16 at 5 24 29 PM" src="https://github.com/forem/forem/assets/9396084/fb31a0ae-f987-4cdd-9693-62ca03c72f1e">

_For more info, check out the
[Forem Accessibility Docs](https://developers.forem.com/frontend/accessibility)._

## Added/updated tests?
